src/script/page/message-list/MentionSuggestions/MentionSuggestions.tsx

```diff
@@ -24,9 +24,9 @@ import {KEY} from 'Util/KeyboardUtil';
 import {clamp} from 'Util/NumberUtil';
 import useEffectRef from 'Util/useEffectRef';
 
-import {useFadingScrollbar} from '../../ui/fadingScrollbar';
-import MentionSuggestionsItem from './MentionSuggestions/MentionSuggestionsItem';
```

Comment:
Oh, this component can be moved as well in the new MentionsSuggestions folder :)